.travis.yml

```diff
@@ -110,7 +110,7 @@ jobs:
         FILE_ENV="./ci/test/00_setup_env_native_multiprocess.sh"
 
     - stage: test
-      name: 'x86_64 Linux  [GOAL: install]  [xenial]  [no wallet]'
+      name: 'x86_64 Linux  [GOAL: install]  [bionic]  [no wallet]'
```

Comment:
in commit ea39d8bc9e1b7c23486abc5b28a20d4660fe93dd:

Sorry this needs a rebase. I tried to avoid that, but it seems `git merge` isn't as smart as `patch` when it comes to line-offsets.

```
$ git show ea39d8bc9e1b7c23486abc5b28a20d4660fe93dd | patch -p1 --dry-run 
checking file .python-version
checking file .travis.yml
Hunk #2 succeeded at 90 with fuzz 1 (offset -20 lines).
checking file ci/test/00_setup_env_native_nowallet.sh
checking file configure.ac
checking file doc/dependencies.md

$ git merge ea39d8bc9e1b7c23486abc5b28a20d4660fe93dd
Auto-merging doc/dependencies.md
Auto-merging configure.ac
Auto-merging .travis.yml
CONFLICT (content): Merge conflict in .travis.yml
Automatic merge failed; fix conflicts and then commit the result.

Reply:
Done